cli/pkg/scanner/export/export.go

```diff
@@ -55,12 +66,41 @@ func createApplicationVulnerabilityScan(m *scanner.MergedResults, layerCommands
 					ResourceType: getResourceType(m),
 					ResourceHash: m.Source.Hash,
 				},
-				ResourceLayerCommands: createResourceLayerCommands(layerCommands),
+				ResourceLayerCommands:     createResourceLayerCommands(layerCommands),
+				CisDockerBenchmarkResults: createCISDockerBenchmarkResults(cisDockerBenchmarkResults),
 			},
 		},
 	}
 }
 
+func createCISDockerBenchmarkResults(results dockle_types.AssessmentMap) []*models.CISDockerBenchmarkCodeInfo {
+	var ret []*models.CISDockerBenchmarkCodeInfo
+
+	for _, info := range results {
+		ret = append(ret, &models.CISDockerBenchmarkCodeInfo{
+			Assessments: createCISDockerBenchmarkAssesment(info.Assessments),
+			Code:        info.Code,
+			Level:       int64(info.Level),
+		})
+	}
+
+	return ret
+}
+
+func createCISDockerBenchmarkAssesment(assesments dockle_types.AssessmentSlice) []*models.CISDockerBenchmarkAssessment {
```

Comment:
I know it's a typo also in the API, but need to change assesment-> assessment in all the places in export/export_test :)

Reply:
thx